.gitlab-ci.yml

```diff
@@ -218,19 +219,20 @@ gcp-dev:
     - cd gcp/live/dev
     - rake clobber
     - rake configure_serviceaccount_ci_restore
-    - rake
-    - rake test_preferences
-    - rake test_flowmanager
-  after_script:
-    # Clean up even if something failed.
-    - cd gcp/live/dev
-    - rake display_cluster_state
     # We need to chain the following into a single command to prevent various issues
     # caused by subsequent after script commands when destroy task fails.
     # More info: https://issues.gpii.net/browse/GPII-3488
     - (rake destroy && rake destroy_secrets && rake destroy_tfstate[k8s]) || true
     # Terraform state encryption key is also gone with secrets, we have to destroy tfstate as well
     - rake destroy_tfstate[locust] || true
+    - rake || (rake display_cluster_state ; false)
```

Comment:
Why all the `; false` bits?

Reply:
Without the `false`, when `rake` fails but `rake display_cluster_state` succeeds, the entire line succeeds and the build continues. This is not desirable.